src/JavaSession1/ArrayIntersection.java

```diff
@@ -0,0 +1,71 @@
+package JavaSession1;/*Question 5: Find common elements between two arrays.*/
```

Comment:
the package name is not correct. 
1. Shouldn't be started from the capital letter.
2. try to create the subpackage instead of doing camel case. ( Note: not always but wherever you feel right )
"JavaSession1" => java.session1